Makefile

```diff
@@ -153,6 +153,45 @@ data/full-cohort-data.json: data/cohort-data.json data/random-data.json
 	sed '1d' $(word 2,$^) >> $@
 
 
+### COGS Set Up
+
+# The branch name should be the namespace for the new cohort
+BRANCH := $(shell git branch --show-current)
+
+init-cogs: .cogs
+
+templates/$(BRANCH).tsv:
+	echo -e "Term ID\tLabel\tParent Term\tDefinition\tGECKO Category\tSuggested Categories\tComment\nID\tLABEL\tC % SPLIT=|\tA definition\n\tis-required;" > $@
+
+# required env var GOOGLE_CREDENTIALS
+.cogs: | templates/$(BRANCH).tsv
+	cogs init -u $(EMAIL) -t $(BRANCH)
+	cogs add $< -r 2
```

Comment:
This doesn't add anything: there's no value for `$<` because stuff after the `|` does not count.

Reply:
Fixed